File: Selenium/Activities/Activity7.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys

with webdriver.Firefox() as driver:
    
    driver.get("https://v1.training-support.net/selenium/drag-drop")

    actions = webdriver.ActionChains(driver)

    print(driver.title)

    ball = driver.find_element(By.ID, "draggable")
    
    dropzone1 = driver.find_element(By.ID, "droppable")
    dropzone2 = driver.find_element(By.ID, "dropzone2")

    actions.drag_and_drop(ball, dropzone1).perform()
    if(dropzone1.text.find("Dropped!")):
        print("Ball is in dropzone1")

    actions.drag_and_drop(ball, dropzone2).perform()
    if(dropzone2.text.find("Dropped!")):
        print("Ball is in dropzone2")

    driver.close()

File: Selenium/Activities/Activity7.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity7 {

	public static void main(String[] args) {
		
		WebDriver driver =  new FirefoxDriver();
		
		Actions builder = new Actions(driver);
		
		driver.get("https://v1.training-support.net/selenium/drag-drop");
		
		System.out.println(driver.getTitle());
		
		WebElement football = driver.findElement(By.id("draggable"));
        
        WebElement dropzone1 = driver.findElement(By.id("droppable"));
       
        WebElement dropzone2 = driver.findElement(By.id("dropzone2"));
        
        builder.clickAndHold(football).moveToElement(dropzone1).pause(2000).release().build().perform();

        String dropzone1Verify = dropzone1.findElement(By.tagName("p")).getText();
        
        if(dropzone1Verify.equals("Dropped!")) {
            System.out.println("Ball is in dropzone 1");
        }
        
        builder.dragAndDrop(football, dropzone2).build().perform();
     
        String dropzone2Verify = dropzone2.findElement(By.tagName("p")).getText();
        
        if(dropzone2Verify.equals("Dropped!")) {
            System.out.println("Ball is in dropzone 2");
        }
        
        driver.close();

	}

}
